docs/source/data_input.rst

```diff
@@ -3,17 +3,19 @@
 Data Loading and Input
 ======================
 
-Before data can be queried with ``dask-sql``, it needs to be loaded into the dask cluster (or local instance) and registered with the :class:`~dask_sql.Context`.
+Before data can be queried with ``dask-sql``, it needs to be loaded into the Dask cluster (or local instance) and registered with the :class:`~dask_sql.Context`.
 ``dask-sql`` supports all ``dask``-compatible `input formats  <https://docs.dask.org/en/latest/dataframe-create.html>`_, plus some additional formats only suitable for ``dask-sql``.
 
-1. Load it via python
+1. Load it via Python
 ---------------------
 
-You can either use already created dask dataframes or create one by using the :func:`~dask_sql.Context.create_table` function.
-Chances are high, there exists already a function to load your favorite format or location (e.g. s3 or hdfs).
+You can either use already created Dask DataFrames or create one by using the :func:`~dask_sql.Context.create_table` function.
+Chances are high, there exists already a function to load your favorite format or location (e.g. S3 or hdfs).
 See below for all formats understood by ``dask-sql``.
 Make sure to install required libraries both on the driver and worker machines:
 
+..
+    TODO - Incorrectly formatted
 .. tabs::
```

Comment:
Not sure why, but code blocks don't seem to work in tabs (see https://github.com/dask-contrib/dask-sql/blob/932f7067131c4b829aa85d2790b7a79f62dd9910/docs/source/data_input.rst). There are a couple of places where this happens in our docs.